docs/modules/integrate/pages/cdc-connectors.adoc

```diff
@@ -79,20 +187,28 @@ For example, a sink mapping CDC data to a `Customer` class and
 maintaining a map view of latest known email addresses per customer
 (identified by ID) would look like this:
 
-```java
+[source,java]
+----
 Pipeline p = Pipeline.create();
 p.readFrom(source)
  .withoutTimestamps()
  .writeTo(CdcSinks.map("customers",
     r -> r.key().toMap().get("id"),
     r -> r.value().toObject(Customer.class).email));
-```
+----
 
 [NOTE]
 ====
 The key and value functions have certain limitations. They can be used to map only to objects which the Hazelcast member can deserialize, which unfortunately doesn't include user code submitted as a part of the job. So in the above example it's OK to have `String` email values, but we wouldn't be able to use `Customer` directly.
 
 If user code has to be used, then the problem can be solved with the help of the User Code Deployment feature. Example configs for that can be seen in our xref:pipelines:cdc-join.adoc#7-start-hazelcast-jet[CDC Join tutorial].
+====
+
+== Migration Guide
+
+Hazelcast {open-source-product-name} has Debezium CDC connector, however it is based on older version of Debezium.
+While migrating to the new connector should be very simple, there are some things users should remember:
 
-Although User Code Deployment has been deprecated, the replacement User Code Namespaces feature does not yet support Jet jobs or pipelines. For now, continue to use the User Code Deployment solution in this scenario. 
-====
\ No newline at end of file
+ * Package was changed from `com.hazelcast.jet.cdc` to `com.hazelcast.enterprise.jet.cdc`
+ * Artifact names are now `hazelcast-enterprise-cdc-debezium`, `hazelcast-enterprise-cdc-mysql` and `hazelcast-enterprise-cdc-postgres` (instead of `hazelcast-jet-...`).
+ * Debezium replaced all `whitelist`s with `include list`s and `blacklist`s with `exclude list`s, Hazelcast for consistency followed the same naming structure and instead of e.g. `setTableWhitelist` you need to use `setTableIncludeList` method.
```

Comment:
```suggestion
 * Debezium replaced all `whitelist`s with `include list`s and `blacklist`s with `exclude list`s, which we have replicated in our naming; so, for example, use `setTableIncludeList` instead of `setTableWhitelist`.
```
Do we expect users to look at the code or API docs or somewhere else to see/confirm these new names?